reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -867,6 +867,7 @@ public String toString() {
                 + " quantile-high:" + higherQuantile.estimation()
                 + " inter-arrival:" + interArrivalTime.value()
                 + " duration/pending:" + (pending == 0 ? 0 : (double)duration / pending)
+                + " pending:" + pending
```

Comment:
I would *NOT* suggest commans-lang ToStringBuilder, but it would be nice to be consistent.

ClientBuilder(a=1, b=2)
vs
WeightedSocket@123434234 [median: a quantile: b]